sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -234,6 +234,13 @@ class CliSuite extends SparkFunSuite with BeforeAndAfterAll with Logging {
     )
   }
 
+  test("unsupported operations") {
```

Comment:
Yeah, this is the only we can check if CLI will reject the command. This test suite is slow but we can do it for all the unsupported operations.